build/room_ensurer/room_ensurer.py

```diff
@@ -270,6 +276,36 @@ def _create_server_user_power_levels(self) -> Dict[str, Any]:
 
         return server_admin_power_levels
 
+    def _ensure_admin_power_levels(self, room_info: RoomInfo) -> None:
+        log.info(f"Ensuring power levels for {room_info.aliases}")
+        api = self._apis[self._own_server_name]
+        own_user = f"@{self._username}:{self._own_server_name}"
+        supposed_power_levels = self._create_server_user_power_levels()
```

Comment:
Unless I'm reading the code wrong I think this will not work correctly.

`_create_server_user_power_levels()` sets the powerlevel of it's own user to 100 and everyone else to 50. 
This works fine in case of the "first" server creating the room. But here we are using that on all servers which means every server will try to set it's own user's level to 100. 
Since `m.room.power_levels` is set to 100 this will not work for any but the first server.

The question is: Is there are reason to not give the every admin user the same power level? Just because one was first it shouldn't have any special powers IMO.

Reply:
Very well spotted. I was thinking that every admin already has a power level of 100. Maybe I wanted to change this in the past but didnt do it yet. Im gonna change the powerlevels of admins